documentation/cli/04_dependencies.md

```diff
@@ -26,7 +26,7 @@ or
 leo add credits
 ```
 
-If you are deplolying to mainnet, you will need to specify mainnet imports using the `--network` flag as follows:
+If you are deploying to mainnet, you will need to specify mainnet imports using the `--network` flag as follows:
 
 ```
 leo add credits --network mainnet
```

Comment:
Heads up, `leo add` will soon not require the name of the network.
The dependencies in the `program.json` won't require it either.
The Leo compiler will check the dependencies depending which network you are on.